### DISCUSSION TITLE
Possible to have an external event trigger Frigate Event?

### DISCUSSION THREAD

**Discussion by hagak** [blakeblackshear/frigate#4024] [Ask A Question]
So I am very close to replacing my zoneminder setup.  One feature of zoneminder that I currently use a good bit is to trigger it to record an event based on an external trigger.

IE I have Home Assistant call a specific URL in zoneminder to tell it to record "XYZ" event for X seconds.  Use-case is whenever certain doors are opened it starts recording.  This does 2 things for me, it makes sure regardless of motion detection or object detection that a recording happens it also adds metadata to the recording of what triggered it.

Any way to trigger this currently in Frigate?

**Comment by NickM-27** [comment]
It's not currently possible but there is a pinned feature request: https://github.com/blakeblackshear/frigate/issues/1063 and I have a draft implementation: https://github.com/blakeblackshear/frigate/pull/3184

**Reply by geiseri** [reply]
Any news on this?  I used to use onvif2mqtt along with doods2.  Mostly because both cameras use PIR at night.  That seems to work much better than any motion detection I have tried.

**Reply by NickM-27** [reply]
My draft PR is there and more or less done barring the integration work, this feature is not within the focus of 0.12 so may come in the release after that.
